hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/portable/PortableGenericRecord.java

```diff
@@ -41,123 +41,125 @@ public ClassDefinition getClassDefinition() {
         return classDefinition;
     }
 
+    @Nonnull
```

Comment:
Do we need to duplicate these annotations in implementations? They're already present in the interface.

Reply:
IDE gives a warning if you don't. I think it is good to see it also here.

Reply:
I see. Thanks for the clarification.